### ISSUE TITLE
EPUB3/HTML support

### ISSUE THREAD

**Issue by bertfrees** [snaekobbi/pipeline-mod-dedicon#20]
Does Dedicon also intend to use EPUB3 as input format? In that case, mod-dedicon should also have an epub3-to-pef script.

**Comment by dkager** [comment]
Yes, but it’s nto as high a priority as output according to specs. That is, we don’t currently use EPUB3 as a production format, it’s a future plan.
